examples/README.md

```diff
@@ -46,7 +46,7 @@ $ ./bin/example_blas_gemm_usm
 ########################################################################
 
 Running BLAS GEMM USM example on CPU device.
-Device name is: Intel(R) Core(TM) i7-6770HQ CPU @ 2.60GHz
+Device name is: Intel(R) Core(TM) i7-6770HQ Processor @ 2.60GHz
```

Comment:
This is meant to be the output of some examples running. The examples may output "CPU" depending on the device used. If this is an issue should we consider not copying the example outputs here?

Reply:
This is a good point, and after taking a look I agree on not copying the outputs. I've deleted the outputs and kept the relevant showcases of how to actually run the examples. Since there is quite a lot of repetition/overlap in showing how to run examples, I only kept the showcases of one domain to decrease clutter as I didn't see anything specific that would be lost in this. I adjusted the preface text to match these changes. Let me know if something looks off / is missing